compiler/rustc_lint/src/dangling.rs

```diff
@@ -201,22 +202,23 @@ fn is_temporary_rvalue(expr: &Expr<'_>) -> bool {
 
 // Array, Vec, String, CString, MaybeUninit, Cell, Box<[_]>, Box<str>, Box<CStr>,
```

Comment:
Please, update this comment and, more importantly, the doc comment above that mentions that `[Sync]UnsafeCell::get()` methods are not accounted for.

Reply:
The doc comment on lines 46–48